### DISCUSSION TITLE
Linux alternatives to Staxrip

### DISCUSSION THREAD

**Discussion by johnvick** [staxrip/staxrip#1204] [General]
I have Windows desktop with 12 core Ryzen and RTX 3070Ti and Surface laptop 5 with i5 and Iris XE. 
The former runs NVENC, the latter QSVENC.  I'm happy with the results I get transcoding 1 hour TV rips of about 3-4 GB to HEVC = 1 GB per hour. QSV has the edge IMHO.

I have a Proxmox server on Intel NUC12 12th gen i7 again Iris XE, with Ubuntu LXC running Handbrake in docker container. It does the job, but I cannot get it to match the quality I get with Staxrip on Windows.

Any suggestions? I know there is no Linux version of Staxrip. Reason for the question is I want to automate the encoding jobs as I am away from home a lot.

Thanks, John

**Comment by jessielw** [comment]
I think hybrid is the closest solution. Handbrake could also be an option, but a lot less control compared to stax and hybrid (I haven't used hybrid). 

I plan to make a front end for personal uses on linux eventually, but no telling when I'll have time.
